source/ci_source/providers/GitHubActions.ts

```diff
@@ -163,7 +163,7 @@ export class GitHubActions implements CISource {
   }
 
   get useEventDSL() {
-    return this.event.pull_request !== undefined || this.event.issue !== undefined
```

Comment:
this before was false when the event was "pull_request", then the right check was the contrary of what I actually wrote